File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - Python/desafio-roteiro03-parte04.py
numero_secreto = 7
tentativa = 0

while tentativa != numero_secreto:
    tentativa = int(input("Adivinhe o número: "))

    if tentativa > numero_secreto:
        print("Tente um número menor!")
    elif tentativa < numero_secreto:
        print("Tente um número maior!")

print("Parabéns! Você acertou!")

File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - C++/desafio-roteiro03-parte04.cpp
#include <iostream>
using namespace std;

int main(){

    int numeroSecreto = 7;
    int tentativa = 0;

    while(tentativa != numeroSecreto){

        cout << "Adivinhe o numero: ";
        cin >> tentativa;

        if(tentativa > numeroSecreto){
            cout << "Tente um numero menor!" << endl;
        }
        else if(tentativa < numeroSecreto){
            cout << "Tente um numero maior!" << endl;
        }
    }

    cout << "Parabens! Voce acertou!" << endl;

    return 0;
}